### ISSUE TITLE
Fix depdencies

### ISSUE THREAD

**Issue by krt16s** [openfoodfacts/OpenFoodFacts-androidApp#25]
Highest version for 21.x support libs is 21.0.3, not 21.1.1.

Material-dialogs is no longer in jcenter, but only in jitpack. First replace it with 

````
compile('com.github.afollestad.material-dialogs:commons:0.8.5.7@aar') {
transitive = true
}
````

and you have to add jitpack to the allprojects repo list.

Also it would be good to pull as much dependecies from central repos and dont use jar files at all.

**Comment by LuccoJ** [comment]
Please do consider fixing these issues, as I look forward to having this app in my main repository of applications (F-Droid), which may not be able to offer it without these fixes.

**Comment by itchix** [comment]
Done  Commit 82ba05a5bb9991615a0f174bd249c6c6248c0ff7 closes #25
